src/graphql/AwsProvisionerWorkerTypes.graphql

```diff
@@ -194,6 +194,113 @@ type AwsProvisionerWorkerTypeState {
   summary: AwsProvisionerWorkerTypeSummary
 }
 
+enum AwsProvisionerErrorType {
+  INSTANCE_REQUEST
+  TERMINATION
+}
+
+interface Aws {
+  region: String
+  az: String
+  instanceType: String
+}
+
+type AwsProvisionerErrors implements Aws {
+  region: String!
+  az: String!
+  instanceType: String!
+  code: String!
+  type: AwsProvisionerErrorType!
+  workerType: String!
+  time: DateTime!
+  message: String!
+}
+
+type AwsProvisionerRunning implements Aws {
+  region: String!
+  az: String!
+  instanceType: String!
+  running: Int!
+}
+
+type AwsProvisionerTerminationHealth implements Aws {
+  region: String!
+  az: String!
+  instanceType: String!
+
+  # A count of the instances which were shutdown cleanty.
+  # For the purposes of this API, a clean shutdown is one which was initiated by us.
+  # This includes API shutdowns or workers ending themselves.
+  # It does not mean the actual workload ran successfully, rather that we chose to terminate it.
+  clean_shutdown: Int!
```

Comment:
I'm not sure why these are cased with underscores, but if they come from the API that way, then that is inconsistent. we should normalize this for the site so we deal with camel cased properties and not have to add ESLint overrides because of the underscores in the names.

Reply:
Yeah, they come from an API. I think GraphQL allows you to alias property fields. I'll do that.

Reply:
Aliases are for the query, so you'll just have to do a translation mapping when querying from the data loader or mutating from the resolver.